.travis.yml

```diff
@@ -1,17 +1,33 @@
 language: go
-
 go:
-  - 1.6.3
-  - 1.7
-  - tip
-
+- 1.6.3
+- 1.7
+- tip
 cache:
   directories:
-  - $HOME/.glide
-
+  - "$HOME/.glide"
 install:
-  - go get github.com/Masterminds/glide
-  - glide install --strip-vendor
-
+- go get github.com/Masterminds/glide
+- glide install --strip-vendor
 script:
-  - go test -v $(glide novendor)
+- go test -v $(glide novendor)
```

Comment:
Use `./script/test` and `./script/lint`
... or you can merge them both into `./script/test`.  I split them originally, but I don't do that anymore.